src/overlays/actors/ovl_En_Guruguru/z_en_guruguru.c

```diff
@@ -250,7 +250,7 @@ void func_80BC7068(EnGuruguru* this, PlayState* play) {
         }
         if (this->textIdIndex == 12) {
             SET_WEEKEVENTREG(WEEKEVENTREG_38_40);
-            func_801A3B48(0);
+            Audio_MuteSeqPlayerBgmSub(0);
```

Comment:
```suggestion
            Audio_MuteSeqPlayerBgmSub(false);
```

Looks like you called the parameter isMuted and its a bool?